File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[2 * nums.length];
        int i = 0;
        for (int curr : nums){
            ans[i] = curr;
            ans[i+nums.length] = curr;
            i +=1;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        ans = set(nums)
        switch = sorted(ans)
        lo = 1
        curr = 1
        for i in range(1, len(switch)):
            if switch[i] == switch[i-1] + 1:
                curr += 1
                if curr > lo:
                    lo = curr
            else:
                curr = 1
        return lo